climpred/bootstrap.py

```diff
@@ -340,12 +342,22 @@ def bootstrap_compute(
         results = results.drop(res_drop)
         ci = ci.drop(ci_drop)
     results = xr.concat([results, ci], 'results')
-    results['results'] = [
-        'skill',
-        'p',
-        'low_ci',
-        'high_ci',
-    ]  # add ci levels in metadata
+    results['results'] = ['skill', 'p', 'low_ci', 'high_ci']
+    metadata_dict = dict()
+    metadata_dict['confidence interval levels'] = str(ci_high) + ' ' + str(ci_low)
```

Comment:
Was wondering if it'd be better to initiate all the items at once

metadata_dict = {
    'confidence_inter_levels': ...
    ...
    'p': ...
}

Reply:
Also might be good here to use an f string. Just a style thing I prefer for python3.

`... = f"{ci_high} {ci_low}"`